forge.config.js

```diff
@@ -106,11 +110,21 @@ function notarizeMaybe() {
     return;
   }
 
+  if (process.env.npm_lifecycle_event !== 'make') {
+    console.log(`Not in "make" mode, skipping notarization`);
```

Comment:
I think we only want to do this when we're publishing. Although this will change back once we move to CircleCI.

```suggestion
  if (process.env.npm_lifecycle_event !== 'publish') {
    console.log(`Not in "publish" mode, skipping notarization`);
```